packages/component/src/hooks/useEmojiFromStyles.js

```diff
@@ -0,0 +1,51 @@
+/*eslint no-useless-escape: "off"*/
+/*eslint require-unicode-regexp: "off" */
+import useStyleOptions from './useStyleOptions';
+
+function escapeRegexp(emoticon) {
+  return emoticon.replace(/[\\^$*+?.()|[\]{}]/g, '\\$&');
+}
+
+export default function useEmojiFromStyles() {
+  const [{ emojiAutocorrect: autocorrect, emojiList: customEmojiList }] = useStyleOptions();
+
+  const emojiAutocorrect = autocorrect || false;
+
+  const emojiUnicodeMap = customEmojiList || {
+    ':)': '😊',
```

Comment:
Is it expected not to have `:(` in the default list? The list from Office has it.

Reply:
not sure how I missed this.